File: codeforces/411/A.py
import sys

x = input()

count_small = 0
count_big = 0
for i in x:
      if i.islower():
            count_small += 1
for y in x:
    if y.isupper():
        count_big += 1
u = len(x)

if u >= 5:
    if count_big >= 1:
        if count_small >= 1:
            if "1" in x or "2" in x or "3" in x or "4" in x or "5" in x or "6" in x or "7" in x or "8" in x or "9" in x or "0" in x:
                print("Correct")
                sys.exit()


print("Too weak")

File: codeforces/411/A.cpp
#include <iostream>

using namespace std;


int main()
{

    string s;
    cin >> s;
    int n=0,u=0,l=0;
    for(int i = 0;i<s.size();i++){
        if(isdigit(s[i])) {
            n++;
        }
    }
    for(int i = 0;i<s.size();i++){
        if(isupper(s[i])) {
            u++;
        }
    }
    for(int i = 0;i<s.size();i++){
        if(islower(s[i])) {
            l++;
        }
    }
    if(s.size() >= 5 && u >= 1 && n >= 1 && l >= 1) {
        cout << "Correct";
    }
    else {
        cout << "Too weak";
    }
}